src/main/java/ru/testtask/controller/WebSocketEventListener.java

```diff
@@ -0,0 +1,74 @@
+package ru.testtask.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.EventListener;
+import org.springframework.messaging.simp.SimpMessageSendingOperations;
+import org.springframework.messaging.simp.stomp.StompHeaderAccessor;
+import org.springframework.stereotype.Component;
+import org.springframework.web.socket.messaging.SessionConnectedEvent;
+import org.springframework.web.socket.messaging.SessionDisconnectEvent;
+import ru.testtask.model.Message;
+
+import javax.annotation.PostConstruct;
+import java.security.Principal;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.String.format;
+
+@Slf4j
+@Component
+public class WebSocketEventListener {
+
+    private final List<String> usernames = new ArrayList<String>();
```

Comment:
`usernames` is list with unique values, maybe it is better to use `Set` instead of `List`